modules/microshift-get-cluster-id-kubesystem.adoc

```diff
@@ -6,11 +6,11 @@
 [id="microshift-get-cluster-id-kubesystem_{context}"]
 = Getting the cluster ID of a running cluster
 
-For a cluster that is running, you can use either step1 or step2 to get the cluster ID.
+Use either the of the following steps to get the ID of a running cluster.
 
 .Procedure
 
-. Get the cluster ID of the cluster that is running by entering the following command:
+* Get the ID of a running cluster by using `oc get` by entering the following command:
```

Comment:
Attempting to make it consistent with the other statements.
```suggestion
* Get the ID of a running cluster by retrieving the `kube-system` namespace UID by entering the following command:
```

Reply:
Resolving without change as discussed on Slack.